### DISCUSSION TITLE
Will hubs be alive in 2021?

### DISCUSSION THREAD

**Discussion by ocularvr** [mozilla/hubs#2982] [General Questions & Discussion] [answered]
I am very curios what the users views are. Mozilla has reduced its work force drastically in August and the errors and lack of documentation for hubs is making me scratch my head. Things like the fact that no one from Mozilla really looks into any posts or has any interest to indicate which direction we should go is worry some to say the least. On one hand it is claimed that Hubs will continue, on the other I cannot understand why there is such a huge lack of documentation and people. What do you think?

**Comment by Letitbe133** [comment]
It is a concern for me too... New to Hubs but I find it's a great tool and really looking forward to keep using it. I guess they have lots of think to do and think about right now and that they're trying to figure out a safe way to keep it alive and improve it

**Comment by rawnsley** [comment]
When the layoffs were announced Mozilla stated that Hubs would be one of the projects that would continue. In fact they appear to be increasing their focus on Hubs having recently added several new members to the team.

My experience has been that the team are reasonably responsive to support requests despite a difficult period: COVID, the departure of a lead engineer, and the downsizing of Mozilla itself. They are certainly more responsive than many premium products I have used in the past.

Personally I would focus a little more on stability at the cost of new features. It is a difficult trade-off because there as many feature requests as their are users and the competition is heating up. Saying no to new features is the hardest part of product management and particularly difficult in such a democratic environment, but it is always easy to be an armchair product manager :-)

It is worth observing that as Hubs is open source it can never really die and I can recommend @gfodor's article [The Path To Mozilla Hubs](https://medium.com/@gfodor/the-path-to-mozilla-hubs-2697e635490d), which touches on this subject following his experiences at AltSpace.

**Reply by ocularvr** [reply]
Great article, thanks for sharing.

**Comment by ocularvr** [comment]
Bump Mozilla team!!!!

**Comment by emclaren** [comment]
Hi folks, as mentioned in the recent blog post (https://blog.mozvr.com/update-on-mozilla-mixed-reality-2/) Mozilla is focusing its efforts on Hubs & Hubs Cloud. We are increasing our investment in the project, and our team is growing. We are working hard on some exciting updates that will make Hubs even better. 

The last few months have been exceptionally busy for our team, and it's true that during this time the documentation hasn't always stayed in sync with all the new features coming out. We've been working hard to fill the gaps (particularly for Hubs Cloud), and respond to questions. 

As we move forward, is there a particular area in the documentation you would like to see improved? Or do you have any specific feedback that you think would improve it? If so, please do let us know on the docs GitHub https://github.com/MozillaReality/hubs-docs we welcome feedback on what to improve under the Issues tab.

**Reply by ocularvr** [reply]
Thank you for the positive feedback! If I could throw in my bit on this, a simple document stating how to rebuild the client on AWS would go a long way for me. Thinking ahead, that would be really useful for many people. Right now, this document is not enough - its a vital subject, I get to half way the  cant go forward after the prerequisites section: https://hubs.mozilla.com/docs/hubs-cloud-custom-clients.html
It would be nice to have a document stating the steps and the requirements.
Again, thank you for your interest.

**Reply by emclaren** [reply]
Created an issue for this over on the hubs docs page: https://github.com/MozillaReality/hubs-docs/issues/68

If you have any other feedback we would be happy to hear it. It would be best to have it on the Hubs Docs github page though so we can track it as we do updates to the docs site.

**Reply by NagarajuSamudrala** [reply]
Hello,

For me, focus on documentation can be improved for two items.
1. Compilation instructions for production environment. As of now, only instruction for local compilation is present.
2. Instructions on Integrating Dialog with reticulum can be improved.

Please help with up to date documentation

Thanks,
Nagaraju Samudrala

**Comment by emclaren** [comment, marked as answer]
yes

![HNY22](https://user-images.githubusercontent.com/6835324/147858475-d832a7bd-0daf-4451-96c1-868035f50341.png)
